test/protocol-tests-core/src/main/resources/software/amazon/awssdk/protocol/suites/cases/smithy-rpcv2-input.json

```diff
@@ -0,0 +1,560 @@
+[
```

Comment:
Is it in the plan to move to smithy's protocol test suite?

Reply:
I think that's the long term plan. Meanwhile this was a one-off translation of the Smithy protocol tests.

Reply:
How long-term?